src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -203,14 +207,15 @@ export function CurrencySearch({
             {({ height }) => (
               <CurrencyList
                 height={height}
-                currencies={filteredSortedTokensWithETH}
+                currencies={disableNonToken ? filteredSortedTokens : filteredSortedTokensWithETH}
```

Comment:
@W3stside nothing from this PR, but I think we can take a look to one issue we had, where we don't show wxDAI and WETH on top of the search token popopup:
![image](https://user-images.githubusercontent.com/2352112/132844296-5216155b-b8b7-4650-91a5-2cc13d41a928.png)


I think is probably here where we need to touch

Reply:
we have a modded version that should be working properly, maybe importis broken